config/allowed_licenses.json

```diff
@@ -10,10 +10,10 @@
     },
     {
       "moduleName": "org.xerial:sqlite-jdbc",
-      "moduleVersion": "3.7.2",
+      "moduleVersion": "3.42.0.1",
       "moduleLicense": "Apache-2.0",
       "moduleCopyright": "David Crawshaw, 2006",
-      "licenseUrl": "https://github.com/xerial/sqlite-jdbc/blob/sqlite-jdbc-3.7.2/LICENSE.zentus",
+      "licenseUrl": "https://github.com/xerial/sqlite-jdbc/blob/3.42.0.1/LICENSE.zentus",
```

Comment:
This file does not contain the Apache 2.0 license 

Apache 2.0 Licence is located at `https://github.com/xerial/sqlite-jdbc/blob/3.42.0.1/LICENSE`

Reply:
resolved in [9aeb2c1](https://github.com/jenkinsci/ecutest-plugin/pull/690/commits/9aeb2c12c224ff250cc72a24b8015930fb3ba03c)